### PR TITLE
st1 [2061][ADD] hr_employee_view_adj

### PR DESCRIPTION
**Pull request by takuyatokunaga**
[2061](https://www.quartile.co/web?#id=2061&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

**Comment by TomohiroAbe-Quartile** [comment]
Pre-commit failed by line below:
`AttributeError: module 'collections' has no attribute 'MutableSet'`

File: hr_employee_view_adj/views/hr_employee_views.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="hr_kanban_view_employees" model="ir.ui.view">
+        <field name="name">hr.employee.kanban</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.hr_kanban_view_employees" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//div[@class='oe_kanban_details']//field[@name='name']"
```
Comment (yostashiro) [inline hr_employee_view_adj/views/hr_employee_views.xml]: Use `hasclass`.

File: hr_employee_view_adj/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2019-2020 Quartile Limited
```
Comment (yostashiro) [inline hr_employee_view_adj/__manifest__.py]:
```suggestion
# Copyright 2021 Quartile Limited
```

**Review by yostashiro — APPROVED** [review]
Please fix a couple of minor points, and it's good to go. 👍

**Pull request merged**